twisted/topfiles/8760.feature

```diff
@@ -0,0 +1 @@
+Client and server TLS connections now support ChaCha20 ciphers when available from OpenSSL.
```

Comment:
I 100% get that. My main concern is that most users are hopefully not using CertificateOptions directly but are instead using endpoints that configure it for them, and so are likely more interested in the affect on those endpoints.